app/routes/companyInterest/components/CompanyInterestPage.tsx

```diff
@@ -882,139 +935,27 @@ const CompanyInterestPage = (props: Props) => {
-              <p className={styles.label}>*</p>
-            </Flex>
-            <p>{interestText.breakfastTalkDescription[language]}</p>
-            <Field
-              placeholder={interestText.breakfastTalkComment[language]}
-              name="breakfastTalkComment"
-              component={TextEditor.Field}
-              rows={10}
-              className={styles.textEditor}
-            />
-          </div>
-        )}
-        {showBedexComment && (
-          <div className={styles.topline}>
-            <Flex alignItems="center" gap={1}>
-              <h4>{EVENT_TYPES.bedex[language]}</h4>
-              <p className={styles.label}>*</p>
-            </Flex>
-            <p>{interestText.bedexDescription[language]}</p>
-            <Field
-              placeholder={interestText.bedexComment[language]}
-              name="bedexComment"
-              component={TextEditor.Field}
-              rows={10}
-              className={styles.textEditor}
-            />
-          </div>
-        )}
-
-        {showOtherEventComment && (
-          <div className={styles.topline}>
-            <Flex alignItems="center" gap={1}>
-              <h4>{EVENT_TYPES.other[language]}</h4>
-              <p className={styles.label}>*</p>
-            </Flex>
-            <p>{interestText.otherEventDescription[language]}</p>
-            <Field
-              placeholder={interestText.otherEventComment[language]}
-              name="otherEventComment"
-              component={TextEditor.Field}
-              rows={10}
-              className={styles.textEditor}
-            />
-          </div>
-        )}
-
-        {showStartupComment && (
-          <div className={styles.topline}>
-            <Flex alignItems="center" gap={1}>
-              <h4>{EVENT_TYPES.start_up[language]}</h4>
-              <p className={styles.label}>*</p>
-            </Flex>
-            <p>{interestText.startUpDescription[language]}</p>
-            <Field
-              placeholder={interestText.startUpComment[language]}
-              name="startupComment"
-              component={TextEditor.Field}
-              rows={10}
-              className={styles.textEditor}
-            />
-          </div>
-        )}
-
-        {showCompanyToCompanyComment && (
-          <div className={styles.topline}>
-            <Flex alignItems="center" gap={1}>
-              <h4>{EVENT_TYPES.company_to_company[language]}</h4>
-              <p className={styles.label}>*</p>
-            </Flex>
-
-            <p>{interestText.companyToCompanyDescription[language]}</p>
-            <Field
-              placeholder={interestText.companyToCompanyComment[language]}
-              name="companyToCompanyComment"
-              component={TextEditor.Field}
-              rows={10}
-              className={styles.textEditor}
-            />
-          </div>
-        )}
+        {eventCommentFields.map((event, i) => {
+          return (
+            event.active && (
```

Comment:
It makes more sense to filter the list instead. Either here or in the declaration.

Reply:
This code snippet is now updated. Could you check and tell me if filtering is needed now or not.

Reply:
I could try to put `showComment` and the `spyValue` part in the `eventCommentFields` list, but maybe filtering isn't necessary now??